From 9d935795ea5a3294f82fe65cca17c79a7b6589f0 Mon Sep 17 00:00:00 2001
From: Mathieu Parent <math.parent@gmail.com>
Date: Thu, 25 Jun 2020 09:48:04 +0200
Subject: [PATCH] Fix FTBFS / Increase the over-estimation for sparse files

BUG: https://bugzilla.samba.org/show_bug.cgi?id=14418
Signed-off-by: Mathieu Parent <math.parent@gmail.com>
Reviewed-by: Andrew Bartlett <abartlet@samba.org>
Reviewed-by: Douglas Bagnall <douglas.bagnall@catalyst.net.nz>

Autobuild-User(master): Andrew Bartlett <abartlet@samba.org>
Autobuild-Date(master): Tue Aug 25 04:23:19 UTC 2020 on sn-devel-184
---
 lib/ldb/tests/ldb_kv_ops_test.c | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/lib/ldb/tests/ldb_kv_ops_test.c b/lib/ldb/tests/ldb_kv_ops_test.c
index 30adebf1952..9db2212895f 100644
--- a/tests/ldb_kv_ops_test.c
+++ b/tests/ldb_kv_ops_test.c
@@ -1717,8 +1717,11 @@ static void test_get_size(void **state)
 	/*
 	 * The tdb implementation of get_size over estimates for sparse files
 	 * which is perfectly acceptable for it's intended use.
+	 * mipsel, ia64: 9994
+	 * ppc64el, powerpc, ppc64: 13369
+	 * sparc64: 5046
 	 */
-	assert_in_range(size, 2500, 5000);
+	assert_in_range(size, 2500, 15000);
 #endif

 	/*
@@ -1746,8 +1749,11 @@ static void test_get_size(void **state)
 	/*
 	 * The tdb implementation of get_size over estimates for sparse files
 	 * which is perfectly acceptable for it's intended use.
+	 * mipsel, ia64: 9994
+	 * ppc64el, powerpc, ppc64: 13369
+	 * sparc64: 5046
 	 */
-	assert_in_range(size, 2500, 5000);
+	assert_in_range(size, 2500, 15000);
 #endif
 	talloc_free(tmp_ctx);
 }
--
GitLab
